src/imageio/format/avif.c

```diff
@@ -183,6 +190,8 @@ void init(dt_imageio_module_format_t *self)
                                 quality,
                                 int);
 #endif
+
+  self->ready = TRUE;
```

Comment:
Remove this change as TRUE is the default.
Same for all other changed below.

Reply:
Done.